docs/release-notes/2.40.0.md

```diff
@@ -110,6 +110,8 @@ VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHA
 - [PMM-12146](https://jira.percona.com/browse/PMM-12146) - Dead Tuples graph on PostgreSQL Vacuum Monitoring Dashboard displayed invalid percentage. The issue has been resolved now.
 - [PMM-12448](https://jira.percona.com/browse/PMM-12448) - Can't start backup if artifact with empty `service_id` exists.
 
-## Known issue
+## Known issues
 
 - [PMM-12517](https://jira.percona.com/browse/PMM-12517) - If you have set custom dashboards as your Home dashboard in version 2.26.0, they may be lost after upgrading to version 2.40.0. To prevent this, we recommend that you back up your custom dashboard before upgrading and recover it after the upgrade.
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - We do not recommend updating to version 2.40.0 due to a critical issue during the migration from SQLite to PostgreSQL. Instead, we suggest waiting for version PMM 2.40.1 to be released.
```

Comment:
can we make it big bold warning in the beginning of RN?

Reply:
ok